index.html

```diff
@@ -49,5 +52,144 @@ <h3>I am a Software Developer</h3>
+					<li><a href="">css</a></li>
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+			</div>
+			<button class="btn">See Project</button>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2">See Project</button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2">See Project</button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2">See Project</button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2">See Project</button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2">See Project</button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2">See Project</button>
+			</div>
+		</div>
+	</section>
+
+	<section class="about-me">
+		<div class="container2">
+			<h1>About Me</h1>
```

Comment:
- [ ] Concerning proper use of HTML semantics. kindly check my detailed review concerning using only one `h1` tag in a project on line `56` above 👍